src/pages/contributing/get-started.mdx

```diff
@@ -0,0 +1,164 @@
+[UI icons](https://www.ibm.com/design/language/iconography/ui-icons/contribute),
+[app icons](https://www.ibm.com/design/language/iconography/app-icons/production),
+and
+[pictograms](https://www.ibm.com/design/language/iconography/pictograms/contribute)
+on the IBM Design Language site.
+
+## The process
+
+Whether you're sharing a proposal or contributing code, designs, and
+documentation, there's a process to follow. Here's what it looks like:
+
+### Volunteer for existing work
+
+One way to volunteer is to look through
+[existing GitHub issues](https://github.com/carbon-design-system/carbon/issues?q=is%3Aopen+is%3Aissue+label%3A%22community+contribution%22)
+labeled with `community contribution`. You can also filter by role:
+[developer](https://github.com/carbon-design-system/carbon/issues?q=is%3Aopen+is%3Aissue+label%3A%22community+contribution%22++label%3A%22role%3A+dev+%F0%9F%A4%96%22_)
+or
+[designer](https://github.com/carbon-design-system/carbon/issues?q=is%3Aopen+is%3Aissue+label%3A%22community+contribution%22++label%3A%22role%3A+design+%3Apencil2%3A%22+).
+Put a comment in the issue saying you'd like the help.
+
+The Carbon team is also happy to help you find a good first issue. Just stop by
+our [office hours](https://carbondesignsystem.com/whats-happening/meetups/) or
+send us a slack message.
+
+### Submit an idea
+
+Have a new idea that you think would benefit Carbon? Or do you need to report a
+bug? First, be sure to look through the
+[issue backlog](https://github.com/carbon-design-system/carbon/issues) to make
+sure it is a novel idea or bug. Then, file your proposal on GitHub using the
+[issue templates](https://github.com/carbon-design-system/carbon/issues/new/choose).
+If you're willing to work on this idea yourself, be sure to let is know in your
+issue! Your idea will then go through a triage process by the Carbon team.
+
+### Iterate and get feedback
+
+Most contributors work in groups of 2-3 and either set up weekly sessions or
+join [meetups](https://carbondesignsystem.com/whats-happening/meetups/) such as
+the Figma Guild, Data Viz Guild, or office hours. In these sessions, it is
+common to share work in progress and ask lots of questions. As you make
+progress, update your GitHub issue.
+
+### Review and published
+
+Most work that results in code will be reviewed directly in a pull request.
+Maintainers will be reviewing your work and making comments, asking questions
+and suggesting changes to be made before they merge your code. When you need to
+make a change, commit and push to your branch normally. Once all revisions to
+your pull request are complete, a maintainer will squash and merge your commits
+for you.
+
+For work that does not have a code deliverable, such as revisions to Figma
+libraries, ask the Carbon team how to finish your contribution.
+
+### Contributor License Agreement
+
+The Carbon core team works for IBM. To accept contributions, we need a signed
```

Comment:
```suggestion
The Carbon core team is part of IBM. To accept contributions, we need a signed
```

Not sure my suggestion is better, but "works" for makes it feel like we are outside contractors.